src/sql/base/browser/ui/panel/media/panel.css

```diff
@@ -70,15 +70,15 @@ panel {
 	font-weight: 600;
 }
 
-.tabbedPanel .tabList .tab-header:hover:not(.active) {
+.tabbedPanel .tabList .tab-header:hover:not(.selected) {
```

Comment:
for historical reason, we have 2 tabbedPanel implementations, panel.ts and panel.component.ts(Angular version), I believe they share some styles in this file.

please test and make sure both of them are working fine.

panel.ts: connection dialog, table designer, etc use this implementation.
panel.component.ts: server/database dashboard uses this implementation.